lib/ingestor-api/runtime/src/ingestor.py

```diff
@@ -88,18 +90,16 @@ def load_into_pgstac(creds: DbCreds, ingestions: Sequence[Ingestion]):
             convert_decimals_to_float(i.item)
             for i in ingestions
         ]
-
-        print(f"Ingesting {len(items)} items")
         loading_result = loader.load_items(
             file=items,
             # use insert_ignore to avoid overwritting existing items or upsert to replace
             insert_mode=Methods.upsert,
         )
 
         # Trigger update on summaries and extents
-        collections = set([item.collection for item in items])
-        for collection in collections:
-            loader.update_collection_summaries(collection)
+        #  collections = set([item["collection"] for item in items])
+        #  for collection in collections:
+        #       loader.update_collection_summaries(collection)
```

Comment:
Why this removal? Should we put behind a settings flag?

Reply:
I'd concur with a settings flag for this behavior but I think this would require a more in-depth PR.  Most of these commits were to make the construct usable for our initial testing.  In this case I'd recommend a follow on PR which makes this behavior configurable on `cron` schedule (which will require additional infrastructure).